{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -4,6 +4,7 @@
 """
 # Third Party Stuff
 import environ
+from corsheaders.defaults import default_headers
```

Comment:
This will also go in the if condition else we will have a problem if it's skipped during code generation.

Reply:
Fixed.